Source/WebCore/loader/FrameLoaderTypes.h

```diff
@@ -271,4 +271,13 @@ template<> struct EnumTraits<WebCore::WebGLLoadPolicy> {
     >;
 };
 
+template<> struct EnumTraits<WebCore::ReloadOption> {
```

Comment:
We could move this up after `EnumTraits<WebCore::PolicyAction>` to pseudo-alphabetize the `EnumTraits`, but not strictly necessary.